tendrl/commons/flows/base_flow.py

```diff
@@ -39,17 +38,10 @@ def __init__(
         self.uuid = uuid
         self.parameters = parameters
         self.job = job
-        self.config = config
         self.definitions = definitions
 
+        # Add etcd_client to parameters via child flows
         self.parameters.update({'log': []})
-        etcd_kwargs = {
-            'port': int(self.config.get("commons", "etcd_port")),
-            'host': self.config.get("commons", "etcd_connection")
-        }
-
-        self.etcd_client = etcd.Client(**etcd_kwargs)
-        self.parameters.update({'etcd_client': self.etcd_client})
```

Comment:
I remember we had done this for a specific reason where we needed etcd_client in further jobs created while import cluster flow on other nodes. How we are planning to achieve this now?